src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1166,6 +1166,24 @@ private DateTimeSupport()
 
             public override void ConvertToJavascript(JavascriptConversionContext context)
             {
+                if (context.Node is MethodCallExpression mce &&
+                    mce.Method.DeclaringType == typeof(DateTime) &&
```

Comment:
Let's change things around.
Instead of specializing only for DateTime, let's support any `ToString`. I want to be able to use this for `this.Age.ToString('000')`, for example.
We also want to support passing the culture.

Reply:
That's not something that is supported in javascript "out of the box", it will require writing some custom jint functions that will do that from c#. That's a .net functionality, that I'm not sure we want to "port" to RQL at all, and if we do, it's not a trivial move and we better think it through..